src/main/java/org/broadinstitute/hellbender/tools/genome/SparkGenomeReadCounts.java

```diff
@@ -139,8 +137,12 @@ private void collectReads() {
         logger.info("Creating full genome bins...");
         final long createGenomeBinsStartTime = System.currentTimeMillis();
         final List<SimpleInterval> fullGenomeBins = createFullGenomeBins(binsize);
-        List<Target> fullGenomeTargetCollection = createTargetCollectionFromSimpleInterval(fullGenomeBins);
-        TargetUtils.writeTargetsAsBed(new File(outputFile.getAbsolutePath() + ".bed"), fullGenomeTargetCollection);
+        List<Target> fullGenomeTargetCollection = createTargetListFromSimpleInterval(fullGenomeBins);
+        try (final TargetWriter writer = new TargetWriter(new File(outputFile.getAbsolutePath() + ".targets.tsv"))){
```

Comment:
Can we create a utility class for CLIs and put a method that does this?  I see it being used in more than one place and it clutters up the calling code.